modules/wrapping/pom.xml

```diff
@@ -12,6 +12,15 @@
 	<description>Wraps non-OSGi JARs into OSGi Bundles for use in the Galasa Ecosystem</description>
 	<url>https://galasa.dev</url>
 
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+		<java.version>1.8</java.version>
+		<maven.compiler.source>1.8</maven.compiler.source>
+		<maven.compiler.target>1.8</maven.compiler.target>
```

Comment:
Should these be a more up to date version?

Reply:
Perhaps we should target Java 11 bytecode now yes. Probably was wider than this one place in the code tho. Will see...

Reply:
See https://stackoverflow.com/questions/38882080/specifying-java-version-in-maven-differences-between-properties-and-compiler-p
for background info. Seems there is a better way of specifying this now.

Reply:
OK. I applied that to the wrapping and maven plugin modules.
Checked that bytecode was being compiled to java 11 format rather than Java 1.8
(major version: 0, minor version: 55). https://stackoverflow.com/questions/9170832/list-of-java-class-file-format-major-version-numbers

Reply:
Fixed the set-version.sh script, and the run-locally.sh script. API server works with the web UI on my local machine now.